frontend/src/high-dimensional/ui/Chart.vue

```diff
@@ -40,6 +40,8 @@ export default {
     return {
       width: 900,
       height: 600,
+      regularLabelColor: '#008c99',
+      matchedLabelColor: '#990d00',
```

Comment:
can you use #c23531? they are also red but that is used in scalar chart line

Reply:
Absolutely yes